index.bs

```diff
@@ -10119,6 +10186,15 @@ and its value is an object called the <dfn id="dfn-interface-object" export>inte
 The property has the attributes { \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val> }.
 The characteristics of an interface object are described in [[#interface-object]].
 
+If the [{{LegacyWindowAlias}}] extended attribute was specified on an interface,
```

Comment:
Right now this doesn't specify clearly whether the LegacyWindowAlias should be exposed if its related interface isn't (e.g. because of SecureContext). NamedConstructor is more explicit here: "for every [{{NamedConstructor}}] extended attribute on an [=exposed=] interface, […]". Maybe copy that text?